psq/queue.py

```diff
@@ -64,7 +69,11 @@ def _get_or_create_subscription(self):
         if not subscription.exists():
```

Comment:
So this is sort of a compromise here, it covers all cases where something could go wrong.